src/elm/Page/Community.elm

```diff
@@ -112,7 +112,7 @@ type alias Member =
 -- VIEW
 
 
-view : LoggedIn.Model -> Model -> Html Msg
+view : LoggedIn.Model -> Model -> { title : String, content : Html Msg }
```

Comment:
maybe we should use a type alias for this, and the best place to add it is “Page.elm” what do you think @amiskov?

Reply:
I knew that you will ask about this :) I'll try to explain why I decided not to create a dedicated type alias.

In Elm, we already have very similar [`Browser.Document`](https://package.elm-lang.org/packages/elm/browser/latest/Browser#Document) type for this but with `List (Html msg)` for HTML output:

```elm
type alias Document msg =
    { title : String
    , body: List (Html msg)
    }
```

I actually [tried](https://github.com/cambiatus/frontend/pull/269/commits/26359e2fc82434ae2f8367624f80cf3c9bcf7824) to use `Document` from all `view` functions, but I didn't like [the way it worked](https://github.com/cambiatus/frontend/pull/269/commits/26359e2fc82434ae2f8367624f80cf3c9bcf7824#diff-6e637b873a9a45f19ff29facd323d55dR960) (line 960) with existing code in `Main.elm`, where `Html msg` was used instead of `List (Html msg)`. I also tried to change everything to work with `List (Html msg)` but it was just too much to refactor.

I decided not to create a dedicated type because it would be very similar to `Browser.Document`. I used just a record to indicate that this is intentionally Ad Hoc solution, which will become `Document` later, in `Main.view`. I found that this kind of implementation is used in [`elm-spa-example`](https://github.com/rtfeldman/elm-spa-example/blob/master/src/Page/Profile.elm#L145) and it looked pretty good for me.

Reply:
I see @amiskov, thanks a lot for the explanation!